test/fuzzer/pedro/utf8_invalid_unicode.test

```diff
@@ -1,15 +1,17 @@
 # name: test/fuzzer/pedro/utf8_invalid_unicode.test
 # group: [pedro]
 
-# invalid utf8, unicode value too high
+# unicode value too high
 statement error
 SELECT upper(decode('\xF6\x96\xB0\x85'::BLOB)::VARCHAR);
 ----
+<REGEX>:Conversion Error.*Failure in decode.*blob to UTF8.*
 
-# invalid utf8, unicode value too high
```

Comment:
the comments here can stay as well

Reply:
returned all these comments. Should I remove the comments repeating the error messages?